clients/go/pkg/justification/processor.go

```diff
@@ -51,12 +54,15 @@ func NewProcessor(v Validator) *Processor {
 
 // Process populates the given audit log request with the justification info from the given token.
 // If the token is empty, this function does nothing.
-func (p *Processor) Process(jvsToken string, logReq *api.AuditLogRequest) error {
-	if jvsToken == "" {
+func (p *Processor) Process(ctx context.Context, logReq *api.AuditLogRequest) error {
+	jvsToken, ok := logReq.Context.GetFields()[TokenHeaderKey]
+	jvsTokenStr := jvsToken.GetStringValue()
+	if !ok || jvsTokenStr == "" {
```

Comment:
Potential bug: if ok is `false`, `jvsToken` could be `nil`. Calling `jvsToken.GetStringValue()` would panic in that case.

`ok` should be checked before calling `jvsToken.GetStringValue()`